compatibility-verifier/sample-test-suite/post-broker-rollback.yaml

```diff
@@ -20,6 +20,12 @@
 # Operations to be done.
 description: Operations to be run after broker rollback
 operations:
+  - type: tableOp
```

Comment:
why are we creating this table again and again? We never stop zookeeper, so any table created once remains.

Reply:
Because we want to have a constant numEntriesScanned in each run to validate the indices/partitioning. If we keep pushing to the table the number will grow and hard to validate. Therefore, I delete and re-create this table in each test.

Reply:
We can set the numEntriesscanned to a high value (my recent PR)

Also, I believe that we execute queries for a specific generation number, so at any one run, only one gen num worth of rows will be considered.

Reply:
Yes, set to the highest value across all runs/generations